frigate/storage.py

```diff
@@ -17,6 +24,98 @@
 )
 
 
+class StorageS3:
+    def __init__(self, config: FrigateConfig) -> None:
+        self.config = config
+        if self.config.storage.s3.enabled or self.config.storage.s3.archive:
+            if self.config.storage.s3.endpoint_url.startswith("http://"):
+                try:
+                    session = boto_session()
+                    session.set_config_variable(
+                        "s3",
+                        {
+                            "use_ssl": False,
+                            "verify": False,
+                        },
+                    )
+                    self.s3_client = session.create_client(
+                        "s3",
+                        aws_access_key_id=self.config.storage.s3.access_key_id,
+                        aws_secret_access_key=self.config.storage.s3.secret_access_key,
```

Comment:
You are forcing the use of keys. 

If `config.storage.s3.access_key_id` is NOT set, you should follow the default chain that allows users to use environment variable, `~/.aws/credentials` etc.

See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html